docs/source/howto/data.rst

```diff
@@ -10,7 +10,78 @@ How to work with data
 Importing data
 ==============
 
-`#3994`_
+AiiDA allows users to export data from their database into an export archive file, which can be imported in any other AiiDA database.
+If you have an AiiDA export archive that you would like to import, you can use the ``verdi import`` command (see :ref:`the reference section<reference:command-line:verdi-import>` for details).
+
+.. note:: More detailed information on exporting and importing data from AiiDA databases can be found in :ref:`"How to share data"<how-to:data:share>`.
+
+If, instead, you have existing data that are not yet part of an AiiDA export archive, such as files, folders, tabular data, arrays or any other kind of data, this how-to guide will show you how to import them into AiiDA.
+
+To store any piece of data in AiiDA, it needs to be wrapped in a :py:class:`~aiida.orm.nodes.data.Data` node, such that it can be represented in the :ref:`provenance graph <topics:provenance>`.
+There are different varieties, or subclasses, of this ``Data`` class that are suited for different types of data.
+AiiDA ships with a number of built-in data types.
+You can list these using the :ref:`verdi plugin<reference:command-line:verdi-plugin>` command.
+Executing ``verdi plugin list aiida.data`` should display something like::
+
+    Registered entry points for aiida.data:
```

Comment:
I'm wondering whether it is really necessary to list all data types. From a goal-oriented perspective, it may sufficient to list only a few important ones, but still mention the command.

Reply:
I had already included just a subset, but will reduce even further